static/app/components/modals/inviteMembersModal/useInviteModal.tsx

```diff
@@ -131,11 +133,13 @@ export default function useInviteModal({organization, initialData, source}: Prop
               ? errorResponse.email[0]
               : errorResponse.email;
 
-        const error = emailError || t('Could not invite user');
+        const orgLevelError = errorResponse?.organization || null;
+        const error = orgLevelError || emailError || t('Could not invite user');
 
         setState(prev => ({
           ...prev,
           inviteStatus: {...prev.inviteStatus, [invite.email]: {sent: false, error}},
+          error: orgLevelError ? orgLevelError : undefined,
```

Comment:
```suggestion
          error,
```
Should this be `orgLevelError`? or use the `const error = orgLevelError || emailError || t('Could not invite user');` with the other error types?

Reply:
so `orgLevelError` is the only error we want to show "globally", aka render the `Alert` component. The other errors are for the individual emails themselves, which we don't want to show leverage to trigger the component, since, well, each email has it's own unique error haha. Hopefully that gives some insights, but open to renaming it to give more clarity